pkg/device/device.go

```diff
@@ -0,0 +1,41 @@
+package device
```

Comment:
You're right, this is linux only (the manpage says they're "not specified in POSIX.1, but are present on many other systems"). I saw you added major and minor in the first place, did you change your mind from [#164 (comment)](https://github.com/appc/spec/pull/164#issuecomment-72647937) and now think we should implement them natively?